devel/minipro/Portfile

```
@@ -26,11 +32,12 @@ depends_run         port:srecord
 
 use_configure       no
 
-variant universal {}
+variant universal   {}
+
+build.args-append   PREFIX=${prefix} CC="${configure.cc} [get_canonical_archflags cc]"
 
-build.args-append   CC="${configure.cc} [get_canonical_archflags cc]"
+destroot.args       DESTDIR=${destroot} PREFIX=${prefix}
```

You should not need to add `DESTDIR=${destroot}` to `destroot.args` because MacPorts already puts it in `destroot.destdir` for you. Note that the makefile portgroup can be used to simplify ports like this one that don't have a configure script.
```suggestion
destroot.args       PREFIX=${prefix}
```